tap_mixpanel/transform.py

```diff
@@ -32,7 +32,7 @@ def transform_event_times(record, project_timezone):
     naive_time = datetime.time(0, 0)
     date = datetime.date(1970, 1, 1)
     naive_datetime = datetime.datetime.combine(date, naive_time)
-    beginning_datetime = timezone.localize(naive_datetime)
+    beginning_datetime = pytz.utc.localize(naive_datetime).astimezone(timezone)
```

Comment:
@prijendev pytz module is not available by default in Python 3.9.
Hence, can you please add the requirement in `setup.py -> dev` for the same

Reply:
Already covered in [https://github.com/singer-io/singer-python/blob/master/setup.py](singer-python)